packages/canary-integration/src/__tests__/capability.test.ts

```diff
@@ -65,116 +31,226 @@ test('verifies capability with signed commit', async () => {
   // Create new did:key with capability attached
   const didKeyWithCapability = didKey.withCapability(capability)
   await didKeyWithCapability.authenticate()
+  return didKeyWithCapability
+}
 
-  await deterministicDocument.update({ foo: 'bar' }, null, {
-    asDID: didKeyWithCapability,
-    anchor: false,
-    publish: false,
-  })
+describe('CACAO Integration test', () => {
+  let ipfs: IpfsApi
+  let ceramic: CeramicApi
+  let wallet: Wallet
+  let didKey: DID
+
+  beforeAll(async () => {
+    ipfs = await createIPFS()
+    ceramic = await createCeramic(ipfs)
+    // Create a did:pkh for the user
+    wallet = Wallet.fromMnemonic(
+      'despair voyage estate pizza main slice acquire mesh polar short desk lyrics'
+    )
+    // Create did:key for the dApp
+    const didKeyProvider = new Ed25519Provider(randomBytes(32))
+    didKey = new DID({ provider: didKeyProvider, resolver: KeyDidResolver.getResolver() })
+    await didKey.authenticate()
+  }, 120000)
 
-  expect(deterministicDocument.content).toEqual({ foo: 'bar' })
-}, 30000)
-
-test('does not allow updating if cacao issuer is not document controller', async () => {
-  // Create a did:pkh for the user
-  const wallet = Wallet.fromMnemonic(
-    'despair voyage estate pizza main slice acquire mesh polar short desk lyrics'
-  )
-  const didPkh = new DID({ resolver: PkhDidResolver.getResolver() })
-  // Create a determinstic tiledocument owned by the user
-  const deterministicDocument = await TileDocument.deterministic(ceramic, {
-    deterministic: true,
-    family: 'testCapabilities2',
+  afterAll(async () => {
+    await ipfs.stop()
+    await ceramic.close()
   })
 
-  const streamId = deterministicDocument.id
+  describe('Resources using StreamId', () => {
+    test('can update with streamId in capability', async () => {
+      // Create a determinstic tiledocument owned by the user
+      const deterministicDocument = await TileDocument.deterministic(ceramic, {
+        deterministic: true,
+        family: 'testCapabilities1',
+        controllers: [`did:pkh:eip155:1:${wallet.address}`],
+      })
+      const streamId = deterministicDocument.id
+      const didKeyWithCapability = await addCapToDid(wallet, didKey, `ceramic://${streamId.toString()}`)
 
-  // Create did:key for the dApp
-  const seed = new Uint8Array([
-    69, 90, 79, 1, 19, 168, 234, 177, 16, 163, 37, 8, 233, 244, 36, 102, 130, 190, 102, 10, 239, 51,
-    191, 199, 40, 13, 2, 63, 94, 119, 183, 225,
-  ])
-  const didKeyProvider = new Ed25519Provider(seed)
-  const didKey = new DID({ provider: didKeyProvider, resolver: KeyDidResolver.getResolver() })
-  await didKey.authenticate()
+      await deterministicDocument.update({ foo: 'bar' }, null, {
```

Comment:
include negative test first?  ie try to update just the didKey, without the capability, assert that it fails

Reply:
Same thing as with `can not create new stream without capability`? Yep, that would make sense to have.